### DISCUSSION TITLE
12/06 안승민 블로그 1회차

### DISCUSSION THREAD

**Discussion by smahn9123** [Orm-camp/blog-feedback#48] [동글동글 4기 피드백]
피드백 부탁드립니다:
https://comgu.tistory.com/entry/Python-Class-변수와-Instance-변수-사용시-조심할-점
https://comgu.tistory.com/entry/Python-Iteration의-원리Iterator-Protocol
https://comgu.tistory.com/entry/Python-dict에-대한-iteration-정리feat-lambda-함수

**Comment by appdevs-lee** [comment]
수업에서 언급한 내용을 넘어서 조금 더 분석적으로 개념 및 심화에 대해서 짚어주는게 너무 좋은 것 같아요. 그리고 글 구조가 읽기 편하게 되어 있어서 그 부분도 좋은 것 같아요. 제가 선호하는 글 스타일입니다ㅋㅋㅋ

**Comment by Jungminchae** [comment]
- 문장이 구성이 짧게 작성되어 있어서 읽기가 쉬운 글이네요. 
- 공식 문서를 읽는 듯한 설명이 잘 된 글입니다.
- 중간에 다른 블로그 글로 링크되는 부분 저는 url 인코딩 깨지는게 보기 싫어서 bookmark로 대체하는데 고려해보셔도 좋을 것 같습니다(링크 눌러보기도 더 쉬워요 ㅋㅋ)
- 앞으로도 좋은 글 기대합니다! 첫 글 공유 감사합니다

**Comment by YuSeungBeom** [comment]
글이 깔끔하게 잘 작성되어 있는 거 같아요. 저도 글 쓰는데 참조해서 쓰겠습니다!
